libs/domain/site/menu-item/menu-item.component.ts

```diff
@@ -0,0 +1,45 @@
+import { resolve } from '@spryker-oryx/di';
```

Comment:
Is it makes sense to have this component without designing a menu component? I'm almost sure we gonna have several types of menu with different design and for now there is no way to make it consistent. But probably it require a sync with design team.

Reply:
I think we wanted to get away with the styling and other design things through composition as much as possible. It appears to work in this case.